src/pages/index.js

```diff
@@ -93,10 +93,6 @@ function PolicyEnforcement() {
           <div className="col">
             <h3>Policy enforcement at scale</h3>
           Some adapters support filtered policy management. This means that the policy loaded by Casbin is a subset of the policy in storage based on a given filter. This allows for efficient policy enforcement in large, multi-tenant environments when parsing the entire policy becomes a performance bottleneck.
-
-  To use filtered policies with a supported adapter, simply call the <code>LoadFilteredPolicy</code> method. The valid format for the filter parameter depends on the adapter used. To prevent accidental data loss, the <code>SavePolicy</code> method is disabled when a filtered policy is loaded.
-
-  For example, the following code snippet uses the built-in filtered file adapter and the RBAC model with domains. In this case, the filter limits the policy to a single domain. Any policy lines for domains other than <code>"domain1"</code> are omitted from the loaded policy:
```

Comment:
I deleted the last two paragraphs.
Reason: https://github.com/casbin/casbin-website/pull/289#issuecomment-1152425826